libs/wcm/utils/requestHandlers.js

```diff
@@ -0,0 +1,70 @@
+/* istanbul ignore file */
+// @todo Add test coverage by #4418
+import { formatJsonRpcError, formatJsonRpcResult } from '@json-rpc-tools/utils';
+import { getSdkError } from '@walletconnect/utils';
+import { transactions, cryptography } from '@liskhq/lisk-client';
+import { ERROR_CASES } from '../constants/lifeCycle';
+import { SIGNING_METHODS } from '../constants/permissions';
+
+const signMessage = async (message, wallet) => {
+  const msgBytes = cryptography.digestMessage(message);
+  const signedMessage = cryptography.signDataWithPrivateKey(msgBytes, wallet.summary.privateKey);
+  return { signature: signedMessage };
+};
+
+const signTransaction = async (
+  rawTx, networkIdentifier, schema, wallet,
+) => {
+  const transaction = {
+    moduleID: rawTx.moduleID,
+    assetID: rawTx.assetID,
+    nonce: BigInt(rawTx.nonce),
+    fee: BigInt(rawTx.fee),
+    senderPublicKey: wallet.summary.publicKey,
```

Comment:
let us send whats required, here in this case we just need `publicKey`? we can avoid passing the whole object where its not necessary.

Reply:
Out of the scope of this PR. but I got your point. I'll take care of it in the next PR.